src/services/middlewareServices/AuthenticationMiddlewareService.ts

```diff
@@ -10,63 +11,180 @@ import { AuthError } from "@errors/AuthError"
 
 import jwtUtils from "@utils/jwt-utils"
 
-import { E2E_TEST_EMAIL, E2E_ISOMER_ID } from "@root/constants"
+import { SessionDataProps } from "@root/classes"
+import {
+  E2E_TEST_EMAIL,
+  E2E_ISOMER_ID,
+  CollaboratorRoles,
+} from "@root/constants"
+import { Site, SiteMember, User } from "@root/database/models"
 import { BadRequestError } from "@root/errors/BadRequestError"
 import { SessionData } from "@root/types/express/session"
 
-const E2E_TEST_REPO = config.get("cypress.e2eTestRepo")
+export const E2E_TEST_REPO = config.get("cypress.e2eTestRepo")
+export const E2E_EMAIL_TEST_SITE = {
+  // NOTE: name is a human readable name
+  // but repo is the underlying github reference
+  name: "e2e email test site",
+  repo: "e2e-email-test-repo",
+}
 const E2E_TEST_SECRET = config.get("cypress.e2eTestSecret")
-const E2E_TEST_GH_TOKEN = config.get("cypress.e2eTestGithubToken")
-const E2E_TEST_USER = "e2e-test"
+
+export const E2E_TEST_GH_TOKEN = config.get("cypress.e2eTestGithubToken")
+export const E2E_TEST_USER = "e2e-test"
```

Comment:
Nit: now that we have two users, can we rename this person to `E2E_TEST_GITHUB_USER`

Reply:
sure